lib/resources/sys_info.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+module Inspec::Resources
+  # this resource returns additional system informatio
+  class System < Inspec.resource(1)
+    name 'sys_info'
+
+    desc 'Use the user InSpec system resource to test for operating system properties.'
+    example "
+      describe sysinfo do
+        its('hostname') { should eq 'example.com' }
+      end
+    "
+
+    def hostname
+      os = inspec.os
+      if os.linux?
+        inspec.command('hostname').stdout.chomp
```

Comment:
I'm not familiar with what `$env:computername` returns in powershell.  I know in ohai we do some work to make sure that we only get the "local" part of the hostname (many of systems are configure such that hostname will return the FQDN).  Personally though, I think I like this better (i.e. keeping this info 1-to-1 with the commands of the same name.